app/uk/gov/hmrc/helptosavefrontend/controllers/EmailController.scala

```diff
@@ -76,11 +82,15 @@ class EmailController @Inject() (val helpToSaveService:          HelpToSaveServi
           withEligibleSession (
             {
               case (s, eligibleWithEmail) ⇒
-                Ok(views.html.email.select_email(eligibleWithEmail.email, SelectEmailForm.selectEmailForm, Some(backLinkFromSession(s))))
+                val emailFormWithData = if (s.changingDetails === true) {
```

Comment:
`s.changingDetails` is a Boolean so you should be able to just have `if (s.changingDetails)`

Reply:
ah yes, thanks